examples/bpi-monolith-test-suite/tests/tests.ts

```diff
@@ -0,0 +1,229 @@
+import { BPI } from '../bpi/bpi';
```

Comment:
all tests should have webhooks for CI/CD so on commit we know all tests passed and nothing broke @ognjenkurtic